EasyClangComplete.sublime-settings

```diff
@@ -46,12 +46,18 @@
   //  - "prefix_paths": [<paths>] <-- array of folders that should be
   //                                  appended to CMAKE_PREFIX_PATHS before
   //                                  cmake is run.
+  //
+  // "makefile" entry has an additional option:
+  //  - "makevars": [<makevars>] <-- Names of the make variables to lookup
   "flags_sources": [
     {"file": "CMakeLists.txt"},
     {"file": "compile_commands.json"},
+    {"file": "Makefile"},
     {"file": "CppProperties.json"},
     {"file": "c_cpp_properties.json"},
     {"file": ".clang_complete"},
+
+    {"makevars": [ "INCLUDES", "CPPFLAGS", "CFLAGS", "CXXFLAGS" ]},
```

Comment:
I would suggest to move this *under* struct of `Makefile` just like I do with `CMakeLists.txt` entry.